src/system-health/health_checker/sysmonitor.py

```diff
@@ -0,0 +1,634 @@
+        if result == "done":
+            #logger.log_info("Received systemd service event for {}".format(unit))
+            timestamp="{}".format(datetime.datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S"))
+            msg={"unit": unit,"evt_src":"sysbus","time":timestamp}
+            self.task_notify(msg)
+            return
+
+    #Sub process for listening the systemd event on dbus
+    def subscribe_sysbus(self):
+        import dbus
+        from gi.repository import GObject
+        from dbus.mainloop.glib import DBusGMainLoop
+
+        DBusGMainLoop(set_as_default=True)
+
+        bus = dbus.SystemBus()
+        systemd = bus.get_object('org.freedesktop.systemd1', '/org/freedesktop/systemd1')
+        manager = dbus.Interface(systemd, 'org.freedesktop.systemd1.Manager')
+
+        manager.Subscribe()
+        manager.connect_to_signal('JobRemoved', self.OnJobRemoved)
+
+        loop = GObject.MainLoop()
+        loop.run()
+
+    def task_worker(self):
+        #logger.log_info("Start Listening to systemd bus (pid {0})".format(os.getpid()))
+        update_proc_name('sysmonitor_sysbus')
+        
+        while not self.task_stopping_event.is_set():
+            self.subscribe_sysbus()
+            time.sleep(1)
+
+    def task_run(self):
+        if self.task_stopping_event.is_set():
+            return
+        self.task_process = multiprocessing.Process(target=self.task_worker)
+        self.task_process.start()
+
+    def task_notify(self, msg):
+        if self.task_stopping_event.is_set():
+            return
+        self.task_queue.put(msg)
+        time.sleep(0.001)
+
+    def task_stop(self):
+        self.task_stopping_event.set()
+        self.task_process.join()
+
+
+class Sysmonitor:
+    def system_service(self):
+
+        myQ = mpmgr.Queue()
+        try:
+            monitor_system_bus=MonitorSystemBusTask(myQ)
+            monitor_system_bus.task_run()
+            
+            monitor_statedb_table=MonitorStateDbTask(myQ)
+            monitor_statedb_table.task_run()
+            
+        except Exception as e:
+            logger.log_error("SubProcess-{}".format(str(e)))
+            sys.exit(1)
+
+
+        try:
+            st_db = SonicV2Connector(use_unix_socket_path=True)
+            st_db.connect(st_db.STATE_DB,False)
+            config_db = ConfigDBConnector()
+            config_db.connect()
+        except Exception as e:
+            logger.log_error("Error: Connection to the DB failed {}".format(str(e)))
+            sys.exit(1)
+
+        update_proc_name('sysmonitor')
+        event = 'SERVICE_EVENT'
+        sysready_lock.lock()
+        #This is run once when sysmonitor bootsup
+        check_system_status(event, st_db, config_db)
+        sysready_lock.unlock()
+        
+        from queue import Empty
+        # Queue to receive the STATEDB and Systemd state change event
+        while True:
+            try:
+                msg = myQ.get(timeout=15)
```

Comment:
Please use constant value instead of magic number

Reply:
Done